src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -31,6 +31,7 @@
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
 [assembly: InternalsVisibleTo("OrleansAWSUtils")]
+[assembly: InternalsVisibleTo("OrleansGoogleUtils")]
```

Comment:
@xiazen is correct.  These should not rely on internals if at all possible.  The fact that we've done this in the past is not justification to continue the practice.

Enabling the extensibility of Orleans is one of our goals, so we'd rather fix the extensibility points than continue to allow internal access in this way.  For error codes, please look at how orleansservicebus handles this.  What other internals do you need?